monitoring/grafana/dashboards/jsonnet/grafana_dashboards.jsonnet

```diff
@@ -0,0 +1,73 @@
+local grafana = import '../../../../grafonnet-lib/grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local annotation = grafana.annotation;
+{
+  "radosgw-sync-overview.json":
+    local graphPanelSchema(title, nullPointMode, stack, formatY1, formatY2, labelY1, labelY2, min, fill, datasource) =
+    graphPanel.new(title=title,nullPointMode=nullPointMode,stack=stack,formatY1=formatY1,formatY2=formatY2,labelY1=labelY1,labelY2=labelY2,min=min,fill=fill,datasource=datasource);
+    local addTargetSchema(expr, intervalFactor, format, legendFormat) =
+    prometheus.target(expr=expr,intervalFactor=intervalFactor,format=format,legendFormat=legendFormat);
+    local addTemplateSchema(name, datasource, query, refresh, hide, includeAll, sort) =
+    template.new(name=name,datasource=datasource,query=query,refresh=refresh,hide=hide,includeAll=includeAll, sort=sort);
+    
+    dashboard.new(
+    title='RGW Sync Overview',
+    uid='rgw-sync-overview',
+    time_from='now-1h',
+    refresh='15s',
+    schemaVersion=16,
+    tags=["overview"],
+    timezone='',
+    timepicker={refresh_intervals:['5s','10s','15s','30s','1m','5m','15m','30m','1h','2h','1d'],time_options:['5m','15m','1h','6h','12h','24h','2d','7d','30d']}
+    )
+
+    .addAnnotation(
+    grafana.annotation.datasource(
+        builtIn= 1,
+        datasource= '-- Grafana --',
+        enable= true,
+        hide= true,
+        iconColor= 'rgba(0, 211, 255, 1)',
+        name= 'Annotations & Alerts',
+        type= 'dashboard'
+    )
+    )
```

Comment:
Hm, I didn't know, but now I've looked it up. It seems we can do things like showing Prometheus alerts in Grafana graphs using annotations. We'd need to think about where to put them, because I don't think we want to have them in every panel. But at the right place, that might be really nice. However, that's probably not something to be added in this PR.